sponsors.js

```diff
@@ -69,7 +69,7 @@ function loadGoldSponsors(id, scale) {
     order = genRandArray(numSponsors);
     for (i = 0; i < numSponsors; i++) {
         loadOneGoldSponsor(order[i], elem, scale);
-        if (((i + 1) % 3) == 0) {
+        if (((i + 1) % 4) == 0) {
```

Comment:
Keep in mind `loadGoldSponsors` is shared between both footers and the front page. Changing to 4 in a row also affects the front page.

With 4, now the front page has a weird issue where they won't fit on a single line, so the 4th one is forced onto a new line. And then after the 4th there is a `<br>`, so the second line ends up being having only 1, while the last line still has 3. Like this
![Screenshot 2024-05-08 02 50 45](https://github.com/canadianyoyoassociation/canadianyoyoassociation.github.io/assets/14879236/80f2422a-08ad-444b-bb5e-42aabde30f21)

Reply:
Rather than increasing it to 4, you can either just make the scale smaller for footers, or add a `numPerRow` argument